internal/provider/resource_ably_app.go

```diff
@@ -128,7 +128,14 @@ func (r resourceApp) Read(ctx context.Context, req tfsdk.ReadResourceRequest, re
 
 	// Fetches all Ably Apps in the account. The function invokes the Client Library Apps() method.
 	// NOTE: Control API & Client Lib do not currently support fetching single app given app id
-	apps, _ := r.p.client.Apps()
+	apps, err := r.p.client.Apps()
+	if err != nil {
+		resp.Diagnostics.AddError(
+			"Error reading Resource",
+			"Could not create resource, unexpected error: "+err.Error(),
```

Comment:
The line above says "reading", this one says "create" - I believe this error is triggered by reading and the line above is correct, so this line should say the same.

Reply:
Also not sure why there is different capitalisation in the line above and here - I think they both should either say "Application Resource", or "application resource" - and the same applies to other places where I left comments like this in this PR

Reply:
Will be fixed in #46